Python/Product/VSInterpreters/CPythonInterpreterFactoryProvider.cs

```diff
@@ -32,7 +32,7 @@ class CPythonInterpreterFactoryProvider : IPythonInterpreterFactoryProvider {
         private readonly Dictionary<string, PythonInterpreterInformation> _factories = new Dictionary<string, PythonInterpreterInformation>();
         const string PythonPath = "Software\\Python";
         internal const string FactoryProviderName = "Global";
-        private bool _watchRegistry;
+        internal bool _watchRegistry;
```

Comment:
Ah, you're right. I was going to try and change it from a test but it turned out to be unnecessary. I'll revert